examples/functions/trip/TR_06_response_private_sharing.xml

```diff
@@ -73,9 +73,18 @@
 									<Order>15</Order>
 								</LegAlight>
 								<Service>
+									<OperatingDayRef>2023-08-13</OperatingDayRef>
 									<JourneyRef>1231231231:1231:123123</JourneyRef>
-									<PrivateMode>schoolBus</PrivateMode>
```

Comment:
I think `PrivateMode` doesn't exist any more? Though my feeling is that we should keep it.

Reply:
will check this

Reply:
My feeling came from the fact that "private modes" is spoken of at various places and in the context of the recent #349, see my other comment above. I don't see any other problem.